sdcm/stress_thread.py

```diff
@@ -135,11 +135,20 @@ def create_stress_cmd(self, node, loader_idx, keyspace_idx):
         return stress_cmd
 
     @staticmethod
-    def _add_errors_option(stress_cmd: str, to_add: list):
-        to_add = (' '.join(to_add))
-        if ' -errors' not in stress_cmd:
-            return stress_cmd + ' -errors ' + to_add
-        return re.sub('-errors([ ]+[^-][a-zA-Z0-9-]+)+([ ]* -[a-z]+|[ ]*)', '-errors\\1 ' + to_add + '\\2', stress_cmd)
+    def _add_errors_option(stress_cmd: str, to_add: list) -> str:
+        """
+        Add suboption to -errors option, if such suboption is there, does not add or change it
+        """
+        to_add = list(to_add)
+        current_error_option = next((option for option in stress_cmd.split(' -') if option.startswith('errors ')), None)
+        if current_error_option is None:
+            return stress_cmd + ' -errors ' + ' '.join(to_add)
+        current_error_suboptions = current_error_option.split(' ')[1:]
```

Comment:
It's better to use `.split()` without arguments here. It'll do 2 things: strip and split by any whitespace string.